korman/exporter/material.py

```diff
@@ -166,7 +166,8 @@ def _export_texture_slot(self, bo, bm, hsgmat, slots, idx):
         if slot.use_stencil:
             hsgmat.compFlags |= hsGMaterial.kCompNeedsBlendChannel
             state.blendFlags |= hsGMatState.kBlendAlpha | hsGMatState.kBlendAlphaMult | hsGMatState.kBlendNoTexColor
-            state.clampFlags |= hsGMatState.kClampTexture
+            if slot.texture and slot.texture.type == "BLEND":
+                state.clampFlags |= hsGMatState.kClampTexture
```

Comment:
Is it possible to clamp the stencil texture using other buttons? Looking at the code around the diff, I don't see a way that it happens.